package.json

```diff
@@ -217,12 +217,12 @@
     "semver": "~5.6.0",
     "should": "~13.2.3",
     "sinon": "~7.2.2",
-    "supertest": "~3.3.0",
-    "webpack": "~4.28.2",
-    "webpack-cli": "~3.1.2",
-    "webpack-merge": "~4.1.5",
+    "supertest": "~3.4.2",
+    "webpack": "~4.29.4",
+    "webpack-cli": "~3.2.3",
+    "webpack-merge": "~4.2.1",
     "webpack-stream": "~5.2.1",
-    "yargs": "^12.0.5"
+    "yargs": "~13.2.0"
```

Comment:
All good for yargs https://github.com/yargs/yargs/blob/master/CHANGELOG.md#breaking-changes